pre_commit_hooks/add_task_number/util.py

```diff
@@ -0,0 +1,100 @@
+import io
+import re
+
+from pre_commit_hooks.util import cmd_output
+
+GIT_COMMENT_SECTION_LINE = r"#?[ ]*-+ >8 -+"
+
+
+def get_current_branch() -> str:
+    """Return current branch's name."""
+    return cmd_output("git", "rev-parse", "--abbrev-ref", "HEAD")
+
+
+def retrieve_task(branch: str, branch_regex: str) -> str | None:
+    """Retrieve task from branch according to the given regex."""
+    matches = re.match(branch_regex, branch)
+
+    if not matches:
+        return
+
+    task_number = matches.group("task")
+    return task_number
+
+
+def is_task_in_message(contents: str, task: str) -> bool:
```

Comment:
I suppose this function should check whther task presents in a desired format? I.e. this should be considered as the task presents? 

```
blblallslfsldfsf ABC-123
```

I mean that only lines in a passed task `format` should be considered as added, right?